testsuite/testsuite-jaxrs/src/test/resources/project-application-path.yml

```diff
@@ -0,0 +1,3 @@
+swarm:
+  jaxrs:
+    application-path: /api
```

Comment:
This should follow the format that was defined by https://github.com/wildfly-swarm/wildfly-swarm/pull/394#issuecomment-283896901

@heiko-braun, can you confirm what the actual implemented structure is?